dblookupext/todo.txt

```diff
@@ -0,0 +1,3 @@
+ - Move the interface "HistoryRepository" where it's needed, not in the package where it's implemented
```

Comment:
Mixed feelings about this todo file. Although will help us restructure the work I'm afraid it will be forgotten.

Reply:
Right. Removed file, added `EN-12965`.